certified-connectors/Asite/apiProperties.json

```diff
@@ -0,0 +1,55 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "oauth2",
+          "clientId": "Microsoft Power Automate",
```

Comment:
@asitedeveloper If the client id is dummy one, please keep it as [[DUMMYVALUE]].

Reply:
@sriyen-msft We are assigned dummy value in client id .